addons/udes_stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -1,5 +1,202 @@
-from odoo import models
+from odoo import models, fields, _, api
+from odoo.exceptions import UserError, ValidationError
+import logging
+
+_logger = logging.getLogger(__name__)
 
 
 class StockPickingBatch(models.Model):
     _inherit = "stock.picking.batch"
+
+    user_id = fields.Many2one(
```

Comment:
Q: How does Odoo display this field on the UI? `User`?

I'm sure this has come up before but I can't remember if Odoo is smart enough to remove `id` from the string. :)